node/src/components/contract_runtime.rs

```diff
@@ -858,12 +862,25 @@ impl ContractRuntime {
         engine_state: &EngineState<LmdbGlobalState>,
         metrics: &Metrics,
         trie_or_chunk_id: TrieOrChunkId,
-    ) -> Result<Option<TrieOrChunk>, engine_state::Error> {
+    ) -> Result<Option<TrieOrChunk>, ContractRuntimeError> {
         let correlation_id = CorrelationId::new();
         let start = Instant::now();
-        let result = engine_state.get_trie(correlation_id, trie_or_chunk_id);
-        metrics.get_trie.observe(start.elapsed().as_secs_f64());
-        result
+        let TrieOrChunkId(trie_index, trie_key) = trie_or_chunk_id;
```

Comment:
```suggestion
        let TrieOrChunkId(chunk_index, trie_key) = trie_or_chunk_id;
```

Reply:
Done in [`3da4c1b` (#3255)](https://github.com/casper-network/casper-node/pull/3255/commits/3da4c1be1ed4e3584136bf18848f53685a90e1df)